apps/explorer/src/pages/validators/Validators.tsx

```diff
@@ -1,20 +1,295 @@
+                                        >
+                                            {validatorsStats.lastEpochRewards >
+                                            0
+                                                ? `${validatorsStats.lastEpochRewards}%`
+                                                : '--'}
+                                        </Heading>
+                                    </Stats>
+                                    <Stats
+                                        label="AVG APY"
+                                        tooltip="Average APY"
+                                    >
+                                        <Heading
+                                            as="h3"
+                                            variant="heading2/semibold"
+                                            color="steel-darker"
+                                        >{`${validatorsStats.averageAPY}%`}</Heading>
+                                    </Stats>
+                                </div>
+                            </Card>
+                        )}
+                    </ErrorBoundary>
+                </div>
+
+                <div className="basis-full md:basis-1/2">
+                    <ErrorBoundary>
+                        <Suspense fallback={null}>
+                            <ValidatorMap />
+                        </Suspense>
+                    </ErrorBoundary>
+                </div>
+            </div>
             <div className="mt-8">
                 <ErrorBoundary>
-                    <TopValidatorsCard />
+                    <TableHeader>All Validators</TableHeader>
+                    {isLoading && (
+                        <PlaceholderTable
+                            rowCount={NUMBER_OF_VALIDATORS}
+                            rowHeight="13px"
+                            colHeadings={['Name', 'Address', 'Stake']}
+                            colWidths={['220px', '220px', '220px']}
+                        />
+                    )}
+
+                    {isSuccess && validatorsTableData?.data && (
+                        <>
+                            <TableCard
+                                data={validatorsTableData.data.filter(
+                                    (_, index) =>
+                                        index >=
+                                            (validatorsPageNumber - 1) *
+                                                NUMBER_OF_VALIDATORS &&
+                                        index <
+                                            validatorsPageNumber *
+                                                NUMBER_OF_VALIDATORS
+                                )}
+                                columns={validatorsTableData.columns}
+                            />
+
+                            {validatorsTableData.data.length >
+                                NUMBER_OF_VALIDATORS && (
+                                <Pagination
```

Comment:
@mystie711 I'm curious what we think about removing pagination entirely given this data set has a pretty low upper bound (should never be more than a couple hundred even as we scale out to mainnet).

Reply:
Connected with @Jibz1. Sounds good to me.